plugin-gradle/src/test/java/com/diffplug/gradle/spotless/GradleIntegrationHarness.java

```diff
@@ -231,7 +232,7 @@ static String buildResultToString(BuildResult result) {
 
 	private static File getTestKitDir() {
 		String gradleUserHome = System.getenv("GRADLE_USER_HOME");
-		if (gradleUserHome == null || gradleUserHome.isEmpty()) {
```

Comment:
this is a nifty example and might be a good starter to get in touch with error prone.

@nedtwigg